CommonLibSF/include/RE/B/BSTEvent.h

```diff
@@ -49,7 +49,7 @@ namespace RE
 	public:
 		~BSTEventSource() override = default;  // 00
 
-		void Notify(const Event& a_event)
+		void Notify(void* a_event)
```

Comment:
Shouldn't it still be `const`?

Reply:
Doesn't really matter, it's not meant to be called until the struct is RE'd (each event will need its own `stl::local function` helper I think)